app/assets/v2/js/vue-filters.js

```diff
@@ -90,3 +90,25 @@ Vue.filter('stringReplace', function(activity_type) {
   return activity_names[activity_type];
 });
 
+Vue.filter('truncate', function(text, length, clamp) {
```

Comment:
I've got a version for truncate -> when we rebase this against master
we'll have pick one of those 

https://github.com/gitcoinco/web/pull/6419

cc @octavioamu

Reply:
using yours.